### ISSUE TITLE
QR Code being scanned twice (?)

### ISSUE THREAD

**Issue by kuma48** [32xlevel/iscanner#4]
I use your library to scan an URL and open it via WebView with Intent. When I tried scanning a QR, it's success but the WebView opened twice. Also sometimes, after pressing the back button the WebView was opened again. Is this QR Code being scanned twice? How to scan QR and open WebView only once, not twice?
My code: 
```
scannableCamera.onScanned(object : ScannableCamera.OnScanned {
    @SuppressLint("ResourceType")
    override fun onScanned(barcodes: List<Barcode>) {
        if (barcodes.hasBarcodeType(Barcode.QR_CODE)) {
            barcodes.forEach { barcode -> // Toast all QRs
                if (barcode.isQr()) {
                    if (barcode.isUrl()){
                        val urlValue = barcode.displayValue //url scanned from the qr code
                        val i = Intent(context, ScanFloraActivity::class.java).apply {
                            putExtra("mUrl",urlValue)
                            startActivity(this)
                            flags = Intent.FLAG_ACTIVITY_NO_HISTORY
                        }
                    }
                    } else {
                        activity?.runOnUiThread {
                            Toast.makeText(requireContext(), barcode.displayValue, Toast.LENGTH_SHORT).show()
                        }
                    }
                }
            }
        }
    }
})
```

**Comment by iartr** [comment]
Hello!
1) Please give me QR code that you use, I will try this
2) ScanFloraActivity - is this activity with WebView?

**Comment by kuma48** [comment]
1. QR that I used 
![qr-code (6)](https://user-images.githubusercontent.com/50063885/82317642-cee0ce80-99f8-11ea-9cd1-a9b73c054e25.png)
2. Yes

**Comment by iartr** [comment]
Note that `startActivity()` is **asynchronous**. QR can be read again while the activity is starting.

You may try to invoke `scannableCamera.stopScanning()` in onScanned callback and invoke `scannableCamera.resumeScanning()` if barcode is incorrect. Or you may set a longer delay.

Please try it and tell me the results.

**Comment by iartr** [comment]
I hope your problem was solved. If not, try to take my advice and create new issue.

**Comment by kuma48** [comment]
Yes, it's work when I set a delay. Thank you!
